tests/sizers/wrapsizer.cpp

```diff
@@ -28,74 +28,85 @@
 // test class
 // ----------------------------------------------------------------------------
 
-class WrapSizerTestCase : public CppUnit::TestCase
+class WrapSizerTestCase
 {
 public:
-    WrapSizerTestCase() { }
+    WrapSizerTestCase();
+    ~WrapSizerTestCase();
 
-    virtual void setUp() wxOVERRIDE;
-    virtual void tearDown() wxOVERRIDE;
+protected:
+    wxWindow *m_leftWin;
+    wxSizer *m_leftWrapSizer;
 
-private:
-    CPPUNIT_TEST_SUITE( WrapSizerTestCase );
-        CPPUNIT_TEST( CalcMin );
-    CPPUNIT_TEST_SUITE_END();
-
-    void CalcMin();
-
-    wxWindow *m_win;
-    wxSizer *m_sizer;
-
-    wxDECLARE_NO_COPY_CLASS(WrapSizerTestCase);
+    wxWindow *m_rightWin;
+    wxSizer *m_rightBoxSizer;
+    wxSizer *m_rightWrapSizer;
 };
 
-// register in the unnamed registry so that these tests are run by default
-CPPUNIT_TEST_SUITE_REGISTRATION( WrapSizerTestCase );
-
-// also include in its own registry so that these tests can be run alone
-CPPUNIT_TEST_SUITE_NAMED_REGISTRATION( WrapSizerTestCase, "WrapSizerTestCase" );
+#define TEXT_VALIDATOR_TEST_CASE(name, tags) \
```

Comment:
Assuming we really need this macro at all, it should have the right name:

```suggestion
#define WRAP_VALIDATOR_TEST_CASE(name, tags) \
```